content/en/agent/proxy.md

```diff
@@ -27,7 +27,105 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 ## Web proxy
 
-Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
+### Squid
```

Comment:
We should show what settings govern proxy usage first (user may already have a proxy) so we should probably cover the generic proxy agent config first (`proxy.http`/`proxy.https`)and then show Squid as the easy way to set this up right below it.